.github/workflows/build-install-packages.yml

```diff
@@ -0,0 +1,29 @@
+name: Build Install Packages
+
+on:
+  release:
+    types: [released]
+
+env:
+  VERSION: ${{ github.event.release.tag_name }}
+  PHP_VERSIONS: '7.3 7.4 8.0 8.1'
```

Comment:
Can we use a list?

Reply:
the value can only be string, integer or boolean, and I don't think there's benefit to declaring it as a JSON array only to reconvert it to a bash list.